### PR TITLE
adds larivaar assist support to chromecast

### PR DESCRIPTION
**Pull request by inderpreetsingh**

### CODE REVIEW

File: src/js/index.js [hunk 1]
```diff
@@ -110,6 +110,13 @@ function displayText(json) {
 
   if (prefs['slide-layout']['display-options'].larivaar === true) {
     document.body.classList.add('larivaar');
+    if (prefs['slide-layout']['display-options']['larivaar-assist']) {
```
Comment (tsingh777) [inline src/js/index.js]: think this needs an else. For older versions of desktop app, the vishraams are not showing when larivaar is on.

Comment (inderpreetsingh) [reply src/js/index.js]: so logic in css is 
`if NOT larivaar-assist -> show vishraams` 

Even in older versions there should not be a larivaar-assist class.

Comment (inderpreetsingh) [reply src/js/index.js]: Image from iOS

Comment (tsingh777) [reply src/js/index.js]: You right. Forgot to clear my setting after checking out dev

**Pull request merged**